forum/users/urls.py

```diff
@@ -5,4 +5,6 @@
 
 urlpatterns = [
     path('register/', RegisterView.as_view(), name='register'),
+    path('login/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
+    path('refresh/', TokenRefreshView.as_view(), name='toekn_refresh'),    
```

Comment:
Spelling error in route name
name='toekn_refresh'  # ← there is an error here
Correct to:
name='token_refresh'